packages/analytics-js-integrations/src/integrations/LinkedInInsightTag/browser.js

```diff
@@ -5,6 +5,7 @@ import {
   NAME,
   DISPLAY_NAME,
 } from '@rudderstack/analytics-js-common/constants/integrations/LinkedInInsightTag/constants';
+import { isDefinedAndNotNull } from '@rudderstack/analytics-js-integrations/utils/utils';
```

Comment:
Can you use the utilities from here?
@rudderstack/analytics-js-common/utilities/

Reply:
done